draft-ietf-quic-transport.md

```diff
@@ -1595,11 +1600,11 @@ CONNECTION_CLOSE or APPLICATION_CLOSE frame if it has sufficient state to do so.
 #### Detecting a Stateless Reset
 
 A client detects a potential stateless reset when a packet with a short header
-cannot be decrypted.  The client then performs a constant-time comparison of the
-16 octets that follow the Connection ID with the Stateless Reset Token provided
-by the server in its transport parameters.  If this comparison is successful,
-the connection MUST be terminated immediately.  Otherwise, the packet can be
-discarded.
+either cannot be decrypted or is marked as a potential duplicate.  The client
+then performs a constant-time comparison of the last 16 octets of the packet
+with the Stateless Reset Token provided by the server in its transport
+parameters.  If this comparison is successful, the connection MUST be terminated
```

Comment:
Can you say what "terminated" means? Is it the same as having received CONNECTION_CLOSE?

Reply:
Perhaps "all connection state MUST be discarded."

Reply:
Any of the above seem fine for now; we'll want to tie this in with @janaiyengar's Unified Theory of Connection Closure when that lands.

Reply:
You can't discard state if you are going to enter the draining period.  But we don't need draining, we just need to stop.  (Any subsequent stateless resets we get will get discarded according to our new packet handling rules.)